src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,10 @@
 import React from 'react';
+import { User } from '../../types/types';
 
-export const UserInfo: React.FC = () => (
+type Props = {
+  list: User[];
+};
+
+export const UserInfo: React.FC<Props> = () => (
   <>Put the user here</>
```

Comment:
Replace the placeholder text with actual content. Render the `User` data by mapping through the `users` prop. 🔧